plasmapy/simulation/particle_integrators.py

```diff
@@ -36,6 +36,69 @@ def boris_push(x, v, B, E, q, m, dt):
     dt : `float`
         Timestep, in SI (second) units.
 
+    inplace : 'bool'
+        If True, v and x array are changed with boris_push, default False.
+
+    Returns
+    -------
+    x : `~numpy.ndarray`
+        Particle position x after Boris push algorithm, in SI (meter) units.
+
+    v : `~numpy.ndarray`
+        Particle velocity after Boris push algorithm, in SI (meter/second) units.
+
+    Examples
+    --------
+    >>> B = np.array([[0.0, 0.0, 5.0]])
+    >>> E = np.array([[0.0, 0.0, 0.0]])
+    >>> x_t0 = np.array([[0.0, 0.0, 0.0]])
+    >>> v_t0 = np.array([[5.0, 0.0, 0.0]])
+    >>> x_t1, v_t1 = boris_push(x = x_t0, v = v_t0, B = B, E = E, q = 1.0, m = 1.0, dt = 0.01)
```

Comment:
```suggestion
    >>> x_t1, v_t1 = boris_push(x = x_t0, v = v_t0, B = B, E = E, q = 1.0, m = 1.0, dt = 0.01, inplace=False)
```

we don't technically need to remove `inplace` from the rest.